content/en/docs/tasks/tools/install-minikube.md

```diff
@@ -141,7 +141,7 @@ To install Minikube manually on windows using [Windows Installer](https://docs.m
 
 {{% /capture %}}
 
-## Cleanup everything to start fresh
+## Cleanup local state
```

Comment:
```suggestion
## Cleanup local state {#cleanup-everything-to-start-fresh}
```

so as to preserve inbound hyperlinks that target this page section

Reply:
Thanks for thinking about keeping inbound links valid, @sftim. I agree this is a good change, but I don't want to block a fix for something that people are already reporting issues. I'll make a note to add the anchor back in a future PR.